Update Information Sheet

theck13 opened the pull request:
### Fix
Update color, style, and weight of the header and title text as well as the minimum row height in the ***Information*** bottom sheet.  See the screenshots below for illustration.

![update_information_sheet](https://user-images.githubusercontent.com/3827611/102541011-1867c980-406d-11eb-962f-2262985f776f.png)

### Test
1. Tap any note in list.
2. Tap ***Information*** action in app bar.
3. Notice dimensions as shown in screenshots above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.

theck13 committed:
Update bottom sheet row height from 56dp to 48dp

theck13 committed:
Add minimum height attribute with bottom sheet row height value to header text

theck13 committed:
Update color of header text from accent to note preview attribute

theck13 committed:
Remove text style attribute with bold value from header text

theck13 committed:
Remove text style attribute with italic value from reference list title

theck13 committed:
Add font family attribute with sans serif medium value to reference list title

theck13 committed:
Update header text view from medium to regular

theck13 committed:
Add text size attribute with text bottom sheet value to header text

theck13 committed:
Update reference list title from font family attribute to roboto medium text view

hichamboushaba reviewed (approved):
LGTM, I'll wait to merge once design review has been completed.

SylvesterWilmott commented:
Looks good!

What is the dp height of the reference rows? I am measuring 58dp but it's hard to tell. Can we aim for 64dp total height since that is the recommended height for the condensed list rows [here](https://material.io/components/lists#specs).

SylvesterWilmott commented:
Maybe this is for another PR but looking at this again, it might make sense to hide the reference count unless there are 2 or more references of that note.

theck13 commented:
> What is the dp height of the reference rows? I am measuring 58dp but it's hard to tell. Can we aim for 64dp total height since that is the recommended height for the condensed list rows [here](https://material.io/components/lists#specs).

I tried making the minimum height 64dp, but it didn't change anything.  According to the dimension values, the height is only 58dp (12dp + 12dp + 16sp + 14sp + 2dp + 2dp), but the height doesn't change unless I use a minimum height of 66dp or more.

> Maybe this is for another PR but looking at this again, it might make sense to hide the reference count unless there are 2 or more references of that note.

I would keep it as is just for consistency and ease of scanning visually.  It might not be obvious why there is a reference number for some items and not others.

theck13 committed:
Remove line spacing extra attribute from reference subtitle view

theck13 committed:
Update top and bottom padding of reference list row from 12dp to 16dp

theck13 commented:
I removed the line spacing extra (2dp) and updated the top and bottom padding from 12dp to 16dp.  Is this closer to what we want?

![update_information_sheet_row](https://user-images.githubusercontent.com/3827611/102637652-8c5cad00-4113-11eb-8a1a-dbc5ec5f7d4d.png)

SylvesterWilmott reviewed (approved):
The measurements on the screenshot look good, however on the device the measurements are off. It looks like the information rows didn't change on the actual device. I'm not too concerned about it though, the main goal of the PR was met. Thanks!
